run/src/index.ts

```diff
@@ -133,7 +124,7 @@ function replicateMany<So extends Sources, Si extends Sinks>(
     };
     const error = (err: any) => {
       scheduleMicrotask(() => {
-        logToConsoleError(err);
+        (console.error || console.log)(err);
```

Comment:
Why not throw the error directly?

Reply:
To avoid error swallowing: https://github.com/cyclejs/cyclejs/issues/687#issuecomment-333814900